js/src/indexes.js

```diff
@@ -232,7 +232,7 @@ Indexes.addColumnToIndex = function (sourceArray, arrayIndex, indexChoice, colIn
         $.each(columns, function () {
             columnNames.push($('input[name="field_name[' +  this.col_index + ']"]').val());
         });
-        displayName = '[' + columnNames.join(', ') + ']';
+        displayName = '[' + columnNames.join(', ').trim() + ']';
```

Comment:
```suggestion
        displayName = '[' + columnNames.join(', ').trimRight() + ']';
```

`trimRight` instead of `trimEnd` as it's more compatible.
https://caniuse.com/mdn-javascript_builtins_string_trimend